webhook/webhook_test.go

```diff
@@ -30,7 +30,7 @@ func fakeOptions() Options {
 		DeploymentName:  "events-webhook",
 		ClusterRoleName: "argo-events-webhook",
 		ServiceName:     "webhook",
-		Port:            443,
+		Port:            8443,
```

Comment:
The unit tests were trying to open port 443, which requires the root user to run `make test`. I changed the port to fix this.